contracts/switchboard/native/NativeSwitchboardBase.sol

```diff
@@ -174,21 +170,17 @@ abstract contract NativeSwitchboardBase is ISwitchboard, AccessControl {
      * @param socket_ The address of the remote switchboard.
      * @param chainSlug_ The identifier of the chain the contract is deployed on.
      * @param initiateGasLimit_ The gas limit for executing transactions.
-     * @param executionOverhead_ The overhead for executing transactions.
-     * @param gasPriceOracle_ The address of the gas price oracle.
      */
     constructor(
         address socket_,
         uint32 chainSlug_,
         uint256 initiateGasLimit_,
```

Comment:
remove this

Reply:
removed executionOverhead_, gasPriceOracle_, initiategasLimit_